code/modules/hallucination/mother.dm

```diff
@@ -0,0 +1,81 @@
+/// Your mother appears to scold you.
+/datum/hallucination/your_mother
+	random_hallucination_weight = 2
+	var/obj/effect/client_image_holder/hallucination/your_mother/mother
+
+/datum/hallucination/your_mother/start()
+	var/list/spawn_locs = list()
+	for(var/turf/open/floor in view(hallucinator, 4))
+		if(floor.is_blocked_turf(exclude_mobs = TRUE))
+			continue
+		spawn_locs += floor
+
+	if(!length(spawn_locs))
+		return FALSE
+	var/turf/spawn_loc = pick(spawn_locs)
+	mother = new(spawn_loc, hallucinator, src)
+	mother.AddComponent(/datum/component/leash, owner = hallucinator, distance = get_dist(hallucinator, mother)) //basically makes mother follow them
+	point_at(hallucinator)
+	talk("[hallucinator]!!!!")
+	var/list/scold_lines = list(
+		pick(list("CLEAN YOUR ROOM THIS INSTANT!", "IT'S TIME TO WAKE UP FOR SCHOOL!!")),
+		pick(list("YOU INSULT YOUR GRANDPARENTS!", "USELESS!")),
+		pick(list("I BROUGHT YOU INTO THIS WORLD, I CAN TAKE YOU OUT!!!", "YOURE GROUNDED!!")),
```

Comment:
*you're

Reply:
to be fair i have no clue how i typed the apostrophe in the first line in the first place